clang/test/CodeGen/ptrauth-function.c

```diff
@@ -0,0 +1,28 @@
+// RUN: %clang_cc1 -triple arm64-apple-ios -fptrauth-calls -fptrauth-intrinsics -emit-llvm %s  -o - | FileCheck -check-prefix=CHECK %s
+
+void test_call();
+
+// CHECK-LABEL: define void @test_direct_call()
+void test_direct_call() {
```

Comment:
Are indirect calls tested? I see clang/test/CodeGen/ptrauth-function-lvalue-cast.c testing indirect call via `char *ptr` casted to desired function type, but I don't see a test when we just have a signed function pointer (for example, passed by an argument to a function) and call the function that way. Please let me know if I miss smth.